popup-signed-in-script.js

```diff
@@ -2,4 +2,27 @@ document.querySelector('#sign-out').addEventListener('click', function () {
     chrome.runtime.sendMessage({ message: 'logout' }, function (response) {
         if (response.message === 'success') window.close();
     });
-});
\ No newline at end of file
+});
+
+chrome.extension.onMessage.addListener(function(message, messageSender, sendResponse) {
+    if(message!=null){
+        if(message['type'] == 'searchResp'){
+            tableResp = '<table><thead><tr><th>Track</th><th>Artist</th></tr></thead><tbody>';
+            for(var i=0;i<message['data']['tracks']['items'].length;i++){
+                tableResp += "<tr><td>" + message['data']['tracks']['items'][i]['name'] +"</td>";
+                tableResp += "<td>" + message['data']['tracks']['items'][i]['artists'][0]['name'] +"</td></tr>";
+            }
+            tableResp += "</tbody></table>"
+            resultsPlaceholder.innerHTML = tableResp;
+        }
+    }
+});
+
+var resultsPlaceholder = document.getElementById('results');
+var templateSource = document.getElementById('results-template').innerHTML;
```

Comment:
Hi Saif, are we populating anything here? I don't see the `templateSource` variable being referenced anywhere after assignment

Reply:
Yep you're right. Line 22 is dead code. I initially wanted to use the handlebars library instead of writing the entire table <tr> and <td>, but chrome rejected that library from loading. My bad, please remove that.

Reply:
I also wanted to say that the search API call is simply boilerplate code. I'm guessing you need some kind of web scraper to monitor the webpage and lookup if it finds a song. 

- I explored the youtube option, but that would require us to integrate Google APIs to get information of the video like title and artist from the video URL.   URL retrieval is present in background.js
- An alternative to that is using a scraper, but it seems youtube has some infrastructure preventing us from doing it. Source: [Stackoverflow](https://stackoverflow.com/questions/1216029/get-title-from-youtube-videos)

Reply:
> Yep you're right. Line 22 is dead code. I initially wanted to use the handlebars library instead of writing the entire table tr and td, but chrome rejected that library from loading. My bad, please remove that, as well as lines 50-54 of the HTML file of this script.

Haha no worries! Could you commit the required changes to your branch, i'll merge the  request post that!

Reply:
> I also wanted to say that the search API call is simply boilerplate code. I'm guessing you need some kind of web scraper to monitor the webpage and lookup if it finds a song.
> 
> * I explored the youtube option, but that would require us to integrate Google APIs to get information of the video like title and artist from the video URL.   URL retrieval is present in background.js
> * An alternative to that is using a scraper, but it seems youtube has some infrastructure preventing us from doing it. Source: [Stackoverflow](https://stackoverflow.com/questions/1216029/get-title-from-youtube-videos)

Yup,  was thinking along similar lines! however, we can just extract the title from the browser tab whenever youtube is open instead of scraping the DOM, yeah?